doc/api/stream.md

```diff
@@ -2035,6 +2046,18 @@ added: REPLACEME
   * `signal` {AbortSignal}
 * Returns: {stream.Readable}
 
+### `stream.Readable.isDisturbed(stream)`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `stream` {stream.Readable|ReadableStream}
+* Returns: `boolean`
+
+Returns whether the stream has been read from.
```

Comment:
If this description is accurate, perhaps a better name would just be `stream.Readable.hasRead()` ?

Reply:
It's more to be in line with what the web standards call it.

Reply:
Took the description from https://streams.spec.whatwg.org/#readablestream-disturbed